### PR TITLE
removes some completely shit ambience from the game

### PR DESCRIPTION
**Pull request by Spacedong**
fuck you, stop mirroring shitty /tg/ features and not seeing whether they sound actually fucking decent or not

:cl: Spacedong
del: Removed some recently added ambience sounds.
/:cl:

### CODE REVIEW

File: code/__DEFINES/sound.dm [hunk 1]
```diff
@@ -26,37 +26,25 @@
 								'sound/ambience/ambigen6.ogg','sound/ambience/ambigen7.ogg',\
 								'sound/ambience/ambigen8.ogg','sound/ambience/ambigen9.ogg',\
 								'sound/ambience/ambigen10.ogg','sound/ambience/ambigen11.ogg',\
-								'sound/ambience/ambigen12.ogg','sound/ambience/ambigen14.ogg','sound/ambience/ambigen15.ogg')
```
Comment (GuyonBroadway) [inline code/__DEFINES/sound.dm:29]: NOT

File: code/game/area/Space_Station_13_areas.dm [hunk 2]
```diff
@@ -883,7 +883,6 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 /area/medical
 	name = "Medical"
 	icon_state = "medbay3"
-	ambientsounds = MEDICAL
```
Comment (GuyonBroadway) [inline code/game/area/Space_Station_13_areas.dm:886]: FUCKING!

File: code/game/area/areas/misc.dm [hunk 3]
```diff
@@ -33,7 +33,6 @@
 /area/prison
 	name = "Prison Station"
 	icon_state = "brig"
-	ambientsounds = HIGHSEC
```
Comment (GuyonBroadway) [inline code/game/area/areas/misc.dm:36]: **MODULAR!!**

**Comment by MoreRobustThanYou** [comment]
just turn off your ambience

**Comment by Qustinnus** [comment]
lol spacedong get mad faggot

**Comment by dreamcrab** [comment]
I would rather have Hippie look over this PR and have him personally curate some songs for us, I will talk to him when I get home

**Comment by PopNotes** [comment]
spacedong is an irredeemable retard, remove him from github bls

**Comment by MoreRobustThanYou** [comment]
NOT. MODULAR. also d0ng.

**Pull request closed without merging**